android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -111,17 +111,17 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
         val channel_monitor_bytes = data.write()
         println("ReactNativeLDK: persist_new_channel")
         val params = Arguments.createMap()
-        params.putString("id", byteArrayToHex(id.to_channel_id()))
+        params.putString("id", byteArrayToHex(id.write()))
```

Comment:
So the iOS version is gonna stay `write`, correct?

Reply:
yes, in the end i changed android